content/posts/efficient-navigation-in-vim.md

```diff
@@ -0,0 +1,140 @@
+---
+title: Efficient Navigation in Vim
+date: 2020-07-07
+---
+
+When editing a file, it's quite crucial that one can navigate their cursor around rather quickly. [Vim](https://www.vim.org/) and [NeoVim](https://neovim.io/) allow for many different ways of doing so which, depending on the situation, can be more or less efficient and useful. This article examines the different ways of moving the cursor and compares them. Here, the term of "efficiency" refers to navigating the cursor with as little time and effort (i.e. the number of keystrokes) as possible.
```

Comment:
`crucial that you can navigate your cursor` sounds a bit more relaxed in my opinion.